deployment/src/main/resources/dev-templates/embedded.html

```diff
@@ -0,0 +1,7 @@
+{#if config:property('quarkus.minio.devservices.enabled') != 'false' && config:property('quarkus.minio.url')}
+<a href="{config:property('quarkus.minio.url')}" class="badge badge-light">
```

Comment:
It would be nice to also display user login and password

Reply:
![image](https://user-images.githubusercontent.com/8418431/195972986-52cc4d67-5dfc-40ae-9a03-a0c50a3c5491.png)
like this ?